app/controllers/katello/api/v2/content_view_filter_rules_controller.rb

```diff
@@ -18,7 +18,7 @@ def index_relation
     api :POST, "/content_view_filters/:content_view_filter_id/rules",
         N_("Create a filter rule. The parameters included should be based upon the filter type.")
     param :content_view_filter_id, :number, :desc => N_("filter identifier"), :required => true
-    param :name, Array, of: [String], :desc => N_("package, package group, or docker tag names")
+    param :name, Array, :of => [String, Hash], :desc => N_("package, package group, docker tag names, or hash for RPM bulkloads: [{'name': 'name', 'version': 'version', 'architecture': 'architecture'}]")
```

Comment:
This is a pretty long line (even if it does pass our lenient rubocop rule for line length), can you break this up into 2 (or more) lines?

Reply:
this won't be necessary because I'm not changing that endpoint, and am adding a new one instead.